pwiz_tools/Shared/CommonUtil/SystemUtil/FormUtil.cs

```diff
@@ -20,11 +20,14 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Runtime.InteropServices;
 using System.Windows.Forms;
 using pwiz.Common.Collections;
 
 namespace pwiz.Common.SystemUtil
 {
+    public enum ScrollDirection { horz = 0, vert = 1 }

```

Comment:
The pinvoke website actually uses the existing enum `System.Windows.Forms.Orientation` for this instead of defining a new enum.
https://www.pinvoke.net/default.aspx/user32.setscrollpos